src/main/java/at/hannibal2/skyhanni/data/PurseAPI.kt

```diff
@@ -15,7 +15,7 @@ object PurseAPI {
     private val patternGroup = RepoPattern.group("data.purse")
     private val coinsPattern by patternGroup.pattern(
         "coins",
-        "(Piggy|Purse): §6(?<coins>[\\d,]+).*"
+        "(§.)*(Piggy|Purse): §6(?<coins>[\\d,]+(\\.\\d)?)( ?(§.)*\\([+-](?<earned>[\\w,.]+)\\)?|.*)?$"
```

Comment:
Two things, first cleanup of the regex (see change below. Second why is earned a \\w isn't it a number so \\d
```suggestion
        "(§.)*(Piggy|Purse): §6(?<coins>[\\d,.]+)( ?(§.)*\\([+-](?<earned>[\\w,.]+)\\)?|.*)?$"
```